docs/create_development_environment.md

```diff
@@ -125,7 +125,7 @@ If we will compile our code again:
 npx webpack --config webpack.config.js
 ```
 
-We will see that the generated file content has changed and it is now contains the following code:
+We will see that the generated file content has changed and it now contains the following:
```

Comment:
We will see that the generated file content has changed and now contains the following: